instantlaunches/main.go

```diff
@@ -56,7 +56,7 @@ type InstantLaunch struct {
 	//
 	// Required: true
 	AddedOn string `json:"added_on" db:"added_on"` // formatted timestamp including timezone
-	db      *sqlx.DB
+	// db      *sqlx.DB // not used
```

Comment:
I'm not sure what the deal was here, but it doesn't seem to be used. Could delete the line entirely, but I wanted to ask about it.

Reply:
I'd go ahead and remove it, pretty sure that got missed in some refactoring.